engine/src/main.rs

```diff
@@ -55,9 +54,9 @@ async fn main() {
     env_logger::init();
     let data = load_data().await;
     let db = start_db().await;
-    let repository = repository::new(db);
+    let repository = LaerningToolRepository::new(db);
     repository
-        .batch_create_datasets(
+        .create_batch(
```

Comment:
I want to keep the original naming because we will also allow batch creation of other entities. So it would be great to have differentiation on these batch methods. It is intentional that the repository represents multiple entities. Can you change this back, please?

```suggestion
        .batch_create_datasets(
```

Reply:
The idea was to have a generic naming for methods in the `Repository` trait. I noticed initially the implementation for Game and Dataset both had `batch_create_datasets` and `batch_create_game` hence the idea of `create_batch` takes its type from the type annotation.

I'll revert the naming and just have two methods, so one would only implement methods that suits their needs